lib/Entities/NewsletterEntity.php

```diff
@@ -157,6 +158,15 @@ public function __construct() {
     $this->queues = new ArrayCollection();
   }
 
+  public function __clone() {
+    // reset ID
+    $this->id = null;
+    // reset timestamps
+    $this->setCreatedAt(new DateTimeImmutable());
```

Comment:
This would be a wrong timezone. We shouldn't use the default PHP timezone, we need to set the value based on what is configured in the WordPress: `Carbon::createFromTimestamp($wp->currentTime('timestamp'))`